### DISCUSSION TITLE
[Protokoll] 18.06.2024

### DISCUSSION THREAD

**Discussion by 7ben18** [AdversarialAttacks/.github#39] [Protokolle]
### Wer hat an dieser Sitzung teilgenommen?

- [X] Gabriel Torres Gamez
- [X] Si Ben Tran
- [X] Daniel Perruchoud
- [X] Stephan Heule

### Welche sind die wichtigsten Themen, die an dieser Sitzung besprochen wurden?

- Recap Sitzung von letztem Mal: [Protokoll 13.06.2024](https://github.com/orgs/AdversarialAttacks/discussions/37)
- Vergleich der Validierungs- und Testdaten mittels Feature Maps mit PCA Reduktion
- Administratives

### Wo haben wir Hindernisse? Was blockiert den Fortschritt unseres Projekts?

- Schlechte Metriken auf dem Testdatensatz

### Was wurde an der Sitzung beschlossen?

- Untersuchung der Train-, Val-, Testpartitionierung weiter untersuchen mittels einem Differenzplot. → Berechnung des Mittelwertes pro Pixelfeld von jeder Partition mit anschliessender Heatmap Visualisierung. 
- Problematik mit anderem Modell und explorieren und zeigen, dass dies überall der Case ist, dass der Testdatensatz schlecht performt. 
- Beschreibung das Training, (Hyperparametertuning mit 3 Learningrate und niedrigster Val Loss Checkpoint) im Bericht beschreiben.
- Attacke geschieht durch den Trainingsdatensatz und anschliessende Evaluierung auf Trainings-, Val,- und Testpartition.  → Cross Vergleich möglich. 
- Leseprobe bis am 2. Juli 2024 fertig stellen, Dani und Stephan definieren, welche Kapitel sie lesen solten.

**Comment by 7ben18** [comment]
# Nachtrag von Dani
![image](https://github.com/AdversarialAttacks/.github/assets/66916399/eba6bcb2-eb74-4237-a513-4093b30fa7ec)

**Comment by 7ben18** [comment]
Rohnotizen während der Sitzung

- Gap von Val und Test fuer Covid und Gap von Val und Test fuer Brain
- Testing von brain in ruhe gelassen

- Einfach beschreiben, positiv verhältnis, verteilung untershiede zwischen train und val
- Differenzenplot -> Mittelwert der Pixelwerte und anschliessend die Räumliche Verteilung plotten. 

- Auf andere Modelle zeigen und explorieren

- upvotes -> Kaggle

- overfitten wir wierklich auf VAlidierung?

- Validation Datensatz overfittet, minimaler Checkpoint genommen. 

- Plot muss nicht heissen. 
- Helligkeit der Bilder im Test im Brain

- Ob die Perturbation die Moedlle angreifen können

- Weiters vorgehen: Validierungsdaten fuer Perturbation. 

- Attackiere ich training, val und test. 

- Fokuss: Attackiere Training und schauen was im training oder auch im Validierung passiert.
-
